tests/integration/test_integration_metobs.py

```diff
@@ -269,11 +241,6 @@ def test_integration_metobs_second(
             raw_header_0
             header_0
         """
-        if raw_header_0:
```

Comment:
There is a raw_header still present here as a parameter. Why are we checking it in this test if not in the previous?

Reply:
I honestly don't know. I have just fixed the problems that have turned up.

Reply:
We should probably remove it here too, then.